src/desktop/components/view_in_room/view.coffee

```diff
@@ -20,9 +20,10 @@ module.exports = class ViewInRoom extends Backbone.View
   events:
     'click .js-view-in-room-close': 'remove'
 
-  initialize: ({ @$img, @dimensions }) ->
+  initialize: ({ @imgUrl, @imgSelector, @positionStyles, @dimensions }) ->
```

Comment:
This is cool, but will break the current call. Not sure if that is an issue at all since we scaled the page to 100% new. But as the old one is not deleted yet, theoretically there is a possibility that we might want to fall back.

Reply:
Hmm... I can look at making it a non-breaking change. There's not _too_ many changes, it shouldn't be that bad.